src/main/java/com/robinpowered/sdk/model/Device.java

```diff
@@ -91,10 +92,18 @@ public DateTime getUpdatedAt() {
         return identifiers;
     }
 
+    public Space getSpace() {
+        return space;
+    }
+
     public void setIdentifiers(List<Identifier> identifiers) {
         this.identifiers = identifiers;
     }
 
+    public void setSpace(Space space) {
+        this.space = space;
+    }
```

Comment:
Nit, but we like to group these getters/setters together. You've jumbled them a bit.